.github/workflows/ci.yml

```diff
@@ -1,78 +1,18 @@
 name: CI
+
 on:
   push:
     branches:
       - main
       - 2201.[0-9]+.x
   repository_dispatch:
-    types:
-      check_connector_for_breaking_changes
+    types: check_connector_for_breaking_changes
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-
-      # Set up Java Environment
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-
-      # Setup Ballerina Environment
-      - name: Set Up Ballerina
-        uses: ballerina-platform/setup-ballerina@v1.1.0
-        with:
-          version: latest
-
-      # Grant execute permission to the gradlew script
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
-
-      # Build the project with Gradle
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
-        run: |
-          ./gradlew build -x test
-
-      # Build Ballerina Project
-      - name: Ballerina Build
-        run: bal pack ./ballerina
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-      
-      # Test Ballerina Project
-      - name: Ballerina Test
-        run: bal test --test-report --code-coverage --coverage-format=xml ./ballerina
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-          CONNECTION_STRING: ${{ secrets.CONNECTION_STRING }}
-          QUEUE_NAME: ${{ secrets.QUEUE_NAME }}
-          TOPIC_NAME: ${{ secrets.TOPIC_NAME }}
-          SUBSCRIPTION_NAME1: ${{ secrets.SUBSCRIPTION_NAME1 }}
-          SUBSCRIPTION_NAME2: ${{ secrets.SUBSCRIPTION_NAME2 }}
-          SUBSCRIPTION_NAME3: ${{ secrets.SUBSCRIPTION_NAME3 }}
-
-      - name: Upload coverage reports to Codecov
-        uses: codecov/codecov-action@v3
-
-      - name: Alert notifier on failure
-        if: failure() && (github.event.action == 'check_connector_for_breaking_changes')
-        run: |
-          curl -X POST \
-          'https://api.github.com/repos/ballerina-platform/ballerina-release/dispatches' \
-          --header 'Accept: application/vnd.github.v3+json' \
-          --header 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
-          --data-raw '{
-            "event_type": "notify-ballerinax-connector-build-failure",
-            "client_payload": {
-              "repoName": "module-ballerinax-azure-service-bus",
-              "workflow": "CI"
-            }
-          }'
+  call_workflow:
+    name: Run Connector Build Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-connector-template.yml@main
+    secrets: inherit
+    with:
+      repo-name: module-ballerinax-mongodb
```

Comment:
Should be ASB?

Reply:
Yes. Addressed in [d83f05f](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/d83f05f2a180df4e2fa1dd36ee1b277ff7cda061)